Bug: 26.0.0 build failure with GCC 15

### This is an issue and not a support question which should be asked at https://forum.greenbone.net/?

- [x] This is an issue and not a support question

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

I've applied https://github.com/greenbone/gvmd/commit/37c5d9176cbb16c107f3ce2db34b1dcbb10a0c5e to 26.0.0 but run into the following build failure with GCC 15 (works with GCC 14).

```
[...]
[ 11%] Building C object src/CMakeFiles/all-manage-obj.dir/manage_events.c.o
cd /var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/build/src && /usr/bin/x86_64-pc-linux-gnu-cc -DBINDIR=\"/usr/x86_64-pc-linux-gnu/bin\" -DCACERT=\"/var/lib/gvm/CA/cacert.pem\" -DCA_DIR=\"/var/lib/gvm/gvmd/trusted_certs\" -DCLIENTCERT=\"/var/lib/gvm/CA/clientcert.pem\" -DCLIENTKEY=\"/var/lib/gvm/private/CA/clientkey.pem\" -DCVSS3_RATINGS=0 -DGMP_VERSION=\"22.7\" -DGVMD_CERT_DATABASE_VERSION=8 -DGVMD_DATABASE_VERSION=259 -DGVMD_DATA_DIR=\"/usr/share/gvm/gvmd\" -DGVMD_FEED_DIR=\"/var/lib/gvm/data-objects/gvmd\" -DGVMD_PID_PATH=\"/run/gvmd/gvmd.pid\" -DGVMD_RUN_DIR=\"/run/gvmd\" -DGVMD_SCAP_DATABASE_VERSION=22 -DGVMD_STATE_DIR=\"/var/lib/gvm/gvmd\" -DGVMD_VERSION=\"26.0.0\" -DGVM_CERT_DATA_DIR=\"/var/lib/gvm/cert-data\" -DGVM_CERT_RES_DIR=\"/usr/share/gvm/cert\" -DGVM_DATA_DIR=\"/usr/share/gvm\" -DGVM_FEED_LOCK_PATH=\"/var/lib/gvm/feed-update.lock\" -DGVM_LIB_INSTALL_DIR=\"/usr/x86_64-pc-linux-gnu/lib\" -DGVM_LOG_DIR=\"/var/log/gvm\" -DGVM_NVT_DIR=\"/var/lib/openvas/plugins/\" -DGVM_OS_NAME=\"Linux-6.14.2\" -DGVM_SCAP_DATA_CSV_DIR=\"/var/lib/gvm/scap-data/22\" -DGVM_SCAP_DATA_DIR=\"/var/lib/gvm/scap-data\" -DGVM_SCAP_RES_DIR=\"/usr/share/gvm/scap\" -DGVM_STATE_DIR=\"/var/lib/gvm\" -DGVM_SYSCONF_DIR=\"/etc/gvm\" -DOPENVASD=0 -DOPENVAS_DEFAULT_SOCKET=\"/run/ospd/ospd-openvas.sock\" -DPREFIX=\"/usr/x86_64-pc-linux-gnu\" -DSCANNERCERT=\"/var/lib/gvm/CA/servercert.pem\" -DSCANNERKEY=\"/var/lib/gvm/private/CA/serverkey.pem\" -I/usr/x86_64-pc-linux-gnu/include/gvm -I/usr/x86_64-pc-linux-gnu/include/glib-2.0 -I/usr/x86_64-pc-linux-gnu/lib/glib-2.0/include -I/usr/x86_64-pc-linux-gnu/include/libmount -I/usr/x86_64-pc-linux-gnu/include/json-c -I/usr/x86_64-pc-linux-gnu/lib/openssl-3/include -I/usr/x86_64-pc-linux-gnu/include/blkid -I/usr/x86_64-pc-linux-gnu/include/uuid -I/usr/x86_64-pc-linux-gnu/include/postgresql-17/server -I/usr/x86_64-pc-linux-gnu/include/postgresql/server -march=native -O2 -pipe -Wall -D_BSD_SOURCE -D_ISOC99_SOURCE -D_SVID_SOURCE -D_DEFAULT_SOURCE -D_FILE_OFFSET_BITS=64  -O3 -DNDEBUG -Wformat -Wformat-security -D_FORTIFY_SOURCE=2 -fstack-protector  -fdiagnostics-color=always -MD -MT src/CMakeFiles/all-manage-obj.dir/manage_events.c.o -MF CMakeFiles/all-manage-obj.dir/manage_events.c.o.d -o CMakeFiles/all-manage-obj.dir/manage_events.c.o -c /var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/gvmd-26.0.0/src/manage_events.c

Error:
  * In program cave perform install --hooks --managed-output --output-exclusivity with-others =net-analyzer/gvmd-26.0.0:0::net --destination installed --replacing =net-analyzer/gvmd-26.0.0:0::installed --x-of-y 1 of 1:
  * When installing 'net-analyzer/gvmd-26.0.0:0::net' replacing { 'net-analyzer/gvmd-26.0.0:0::installed' }:
  * When running an ebuild command on 'net-analyzer/gvmd-26.0.0:0::net':
  * Install failed for 'net-analyzer/gvmd-26.0.0:0::net' (paludis::ActionFailedError)

make[2]: Leaving directory '/var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/build'
make[1]: Leaving directory '/var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/build'

/var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/gvmd-26.0.0/src/manage_events.c: In function 'event':
/var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/gvmd-26.0.0/src/manage_events.c:639:5: error: too many arguments to function 'check_tickets'; expected 0, have 1
  639 |     check_tickets (resource_1);
      |     ^~~~~~~~~~~~~  ~~~~~~~~~~
In file included from /var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/gvmd-26.0.0/src/manage_events.c:31:
/var/tmp/paludis/build/net-analyzer-gvmd-26.0.0/work/gvmd-26.0.0/src/manage_sql_tickets.h:53:1: note: declared here
   53 | check_tickets ();
      | ^~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/all-manage-obj.dir/build.make:135: src/CMakeFiles/all-manage-obj.dir/manage_events.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:214: src/CMakeFiles/all-manage-obj.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

Complete build log: [gvmd-26.0.0_gcc15.txt](https://github.com/user-attachments/files/20339979/gvmd-26.0.0_gcc15.txt)

### Expected Behavior

_No response_

### Steps To Reproduce

1. Use GCC 15
2. Try to build gvmd 26.0.0

### Operating System

Exherbo

### Version

_No response_

### Anything else?

_No response_

--- a/src/manage_sql_tickets.h
+++ b/src/manage_sql_tickets.h
@@ -50,7 +50,7 @@ void
 empty_trashcan_tickets ();
 
 void
-check_tickets ();
+check_tickets (task_t);
 
 void
 delete_tickets_user (user_t);

